book/Section-Moving-Beyond.adoc

```diff
@@ -1340,24 +1342,26 @@ play that role. After you have unzipped JanusGraph you will find the `gremlin.sh
 `gremlin.bat` scripts in the `bin` directory under the JanusGraph parent directory. Once
 you have started the Gremlin Console you can, as we will explore in the next few
 sections, tell JanusGraph about the environment you will be operating in in terms of
-back end store and index.
+store backend and index backend.
 
 [[janusinmemory]]
 Using JanusGraph with the 'inmemory' option
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For almost all production use cases, you will be using JanusGraph along with a
-persistent back end store such as Apache Cassandra or Apache HBase. However while
+persistent storage backend such as Apache Cassandra, Apache HBase, Google Cloud BigTable,
+ScyllaDB, Oracle Berkeley DB, DataStax Astra, Amazon Keyspace, etc. However while
 experimenting with JanusGraph it is incredibly useful to be able to get up and
 running quickly without having to worry about configuring all of the back end storage
 components. This is made possible by the 'inmemory' option that JanusGraph
 provides. This essentially allows us to use JanusGraph in the same way as we have
 been using TinkerGraph with all of our graph data stored in the memory of the
 computer. The one big difference however is that JanusGraph, even while using the
 'inmemory' storage model, allows us to experiment with features that TinkerGraph does
-not offer, such as schemas and transactions. We will get into those topics a bit
-later on. First, let's create an instance of JanusGraph from the Gremlin console
-that uses the 'inmemory' storage model.
+not offer, such as schemas and indexing. If you want to host your graph on a single
```

Comment:
TinkerGraph just support creation of an index. It does not support use of an external index.